.bunnyshell/templates/express-react-postgres-helm-k8s/bunnyshell.yaml

```diff
@@ -0,0 +1,106 @@
+  FRONTEND_URL: 'https://frontend-{{ env.base_domain }}'
+  FRONTEND_URL_NO_SCHEME: 'frontend-{{ env.base_domain }}'
+  POSTGRES_DB: bunny_books
+  POSTGRES_PASSWORD: mysecretpassword
+  POSTGRES_USER: postgres
+components:
+  -
+    kind: DockerImage
+    name: frontend-image
+    gitRepo: 'https://github.com/sorinbunnyshell/templates.git'
+    gitBranch: react-books-app/v0.5.0
+    gitApplicationPath: /components/react-books-app
+    context: /components/react-books-app
+    dockerfile: .docker/Dockerfile
+    target: prod
+  -
+    kind: DockerImage
+    name: api-image
+    gitRepo: 'https://github.com/sorinbunnyshell/templates.git'
+    gitBranch: express-rest-api/v0.3.0
+    gitApplicationPath: /components/express-rest-api
+    context: /components/express-rest-api
+    dockerfile: .docker/Dockerfile
+    target: prod
+  -
+    kind: Helm
+    name: postgres
+    runnerImage: 'dtzar/helm-kubectl:3.8.2'
+    deploy:
+      - |
+        cat << EOF > my_values.yaml
+            global:
+                storageClass: bns-network-sc
+            auth:
+                postgresPassword: {{ env.vars.POSTGRES_PASSWORD }}
+                database: {{ env.vars.POSTGRES_DB }}
+        EOF
+      - 'helm repo add bitnami https://charts.bitnami.com/bitnami'
+      - 'helm upgrade --install --namespace {{ env.k8s.namespace }} --post-renderer /bns/helpers/helm/add_labels/kustomize -f my_values.yaml postgres bitnami/postgresql --version 11.9.11'
+      - |
+        POSTGRES_HOST="postgres-postgresql.{{ env.k8s.namespace }}.svc.cluster.local"
+    destroy:
+      - 'helm uninstall postgres --namespace {{ env.k8s.namespace }}'
+    start:
+      - 'kubectl scale --replicas=1 --namespace {{ env.k8s.namespace }} statefulset/postgres-postgresql'
+    stop:
+      - 'kubectl scale --replicas=0 --namespace {{ env.k8s.namespace }} statefulset/postgres-postgresql'
+    exportVariables:
+      - POSTGRES_HOST
+  -
+    kind: Helm
+    name: api
+    gitRepo: 'https://github.com/sorinbunnyshell/templates.git'
+    gitBranch: express-rest-api/v0.3.0
+    gitApplicationPath: /components/express-rest-api
+    runnerImage: 'dtzar/helm-kubectl:3.8.2'
+    deploy:
+      - |
+        cat << EOF > my_values.yaml
+            serviceImage: {{ components.api-image.image }}
+            replicas: 1
+            ingress:
+                className: bns-nginx
+                host: api-{{ env.base_domain }}
+            postgres:
+                host: '{{ components.postgres.exported.POSTGRES_HOST }}'
+                db: '{{ env.vars.POSTGRES_DB }}'
+                user: '{{ env.vars.POSTGRES_USER }}'
+                password: '{{ env.vars.POSTGRES_PASSWORD }}'
+            frontendUrl: '{{ env.vars.FRONTEND_URL }}'
+        EOF
+      - 'helm upgrade --install --namespace {{ env.k8s.namespace }} --dependency-update --post-renderer /bns/helpers/helm/add_labels/kustomize -f my_values.yaml api-{{ env.unique }} ./components/express-rest-api/helm-chart'
```

Comment:
here I am not sure, why this folder is different than the ones from start and stop ./components/express-rest-api/helm-chart vs ./helm/backend